### DISCUSSION TITLE
How to get re-updating of ID3 tags of already recorded tracks to work?

### DISCUSSION THREAD

**Discussion by welles** [jwallet/spy-spotify#482] [Q&A]
Hi everyone! I'm wondering if I need to have a specific set of settings for re-updating of ID3 tags of already recorded tracks to work?
Currently it seems like whatever I do the already existing songs are just skipped without updating the ID3 tags.
Is the setting broken?

**Comment by jwallet** [comment]
https://github.com/jwallet/spy-spotify/issues/489 I will look at it
